specifyweb/frontend/js_src/lib/components/QueryBuilder/helpers.ts

```diff
@@ -376,6 +376,20 @@ const containsRelativeDate = (
       fieldSpec.datePart === 'fullDate'
   );
 
+const containsExpandedStringComparisons = (
```

Comment:
hmm, I though this was already supported in sp6 too but removed by me in 7.6
could you double-check?